feature/experimental/policy/policy_vrf_selection/ate_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -377,27 +383,27 @@ func TestVrfPolicy(t *testing.T) {
 		{
 			desc:      "Match IPv4",
 			policy:    "match-ipv4",
-			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10"},
+			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "Ipv4Vlan20", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20"},
 		},
 		{
 			desc:      "Match IPinIP",
 			policy:    "match-ipip",
-			passFlows: []string{"IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10"},
+			passFlows: []string{"IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20"},
 		},
 		{
 			desc:      "Match IPv4 and IPv6",
 			policy:    "match-ip4ip6",
-			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "Ipv6Vlan20"},
+			passFlows: []string{"Ipv4Vlan10", "IpipVlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan10", "Ipv6Vlan20", "Ipv4Vlan20", "IpipVlan20", "IpipDscp46Vlan20", "IpipDscp42Vlan20", "Ipv6Vlan10"},
 		},
 		{
 			desc:      "Match IPinIP DSCP 46",
 			policy:    "match-ipip-dscp46",
-			passFlows: []string{"IpipDscp46Vlan10"},
+			passFlows: []string{"IpipDscp46Vlan10", "IpipDscp46Vlan20"},
 		},
 		{
 			desc:      "Match IPinIP DSCP 42 or 46",
 			policy:    "match-ipip-dscp42or46",
-			passFlows: []string{"IpipDscp46Vlan10", "IpipDscp46Vlan10"},
+			passFlows: []string{"IpipDscp42Vlan10", "IpipDscp46Vlan10", "IpipDscp42Vlan20", "IpipDscp46Vlan20"},
```

Comment:
Wouldn't adding "IpipDscp42Vlan20", "IpipDscp46Vlan20" fail the test?

Below is the test requirement
 *   Match IPinIP, DSCP 42, 46 - routed to network-instance 10, ensure that DSCP 42 and 46 packets are received at VLAN         
                Ensure that DSCP 0 packets are not received at VLAN 10.

Reply:
Latest changes pushed in. please check. Thanks.